client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -23,12 +23,12 @@ const PaymentMethodItem = ( { method, translate } ) => {
 						</p>
 					)
 				}
-				<p className="payments__method-name">{ method.label }</p>
+				<p className="payments__method-name">{ method.title }</p>
 			</ListItemField>
 			<ListItemField>
-				<p className="payments__method-information">{ method.fee }</p>
+				<p className="payments__method-information">{ method.fees }</p>
```

Comment:
`PaymentMethodItem` will eventually be used by all payment types right? These list items should probably be wrapped in a conditional since `offline` methods don't have most of these props.